modules/app/components/layout/Footer/LandingFooter.tsx

```diff
@@ -0,0 +1,228 @@
+import { Flex, Text, useColorMode } from 'theme-ui';
+import { Icon } from '@makerdao/dai-ui-icons';
```

Comment:
If it's only the footer for the homepage , maybe we can put it under "modules/home"?